test/input/TestFeatureTransfer/synthetic/input/ref.tbl

```diff
@@ -0,0 +1,25 @@
+>Feature gb|AY772699.1|
+<1	99	CDS
+			gene	should-be-output-if-not-ignore_ambig_feature_edge
```

Comment:
I know this is just an artificial feature (the `<1` - `99` CDS) for testing purposes, but it seems weird to have a "gene" tag within a CDS feature. can we turn this into a "product" or something?

Reply:
otherwise, nice battery of edge cases